tools/DocsLinter/Program.cs

```diff
@@ -9,6 +9,9 @@ namespace DocsLinter
 {
   internal class Program
   {
+    private const string GithubRepoBlobPath = "github.com/spatialos/unitygdk/blob";
+    private const string GithubRepoTreePath = "github.com/spatialos/unitygdk/tree";
```

Comment:
Why do we need the `/blob` / `/tree` at the end?

Reply:
Those are two different ways to reference files in the repo. If i just have `github.com/spatialos/unitygdk` it will pick up links to things like the issues board and stuff like that

Reply:
Got it :)

Reply:
Won't this fail for end users that have forked the GDK? Seems like an annoying thing for forkers to have to change.

Reply:
I guess why are forkers changing our docs and running the docs linter?

Reply:
It seems like a valid use-case, if they're modifying/adding things they want to document, and have a build pipeline.

Reply:
The problem is that you can't generically know the repo URL without parsing `git remotes` or similar. We can't guarantee that everyone has the `spatialos/UnityGDK` as `origin` or similar.

We could do a regex? `.*github.com/.*/unitygdk/blob/.*` and similar? But then we enforce the naming of the repo which might be okay.

Reply:
We can generalize further :D `/github\.com\/[^/]+\/[^/]+\/(blob|tree)/`

Reply:
Discussed offline with @jared-improbable. Will add a `.env` file containing a reference to the repo link. This is a simple change for forkers to make if they also want this workflow!

Reply:
Nice :)